app/controllers/admin/move_controller.rb

```diff
@@ -3,44 +3,57 @@
 module Admin
   # Controller for moving resources between collections
   class MoveController < Admin::ApplicationController
-    before_action :set_work
-    before_action :set_status
-    before_action :set_work_form_from_cocina
-
     def new
       authorize!
 
-      @collections = Admin::MoveTargetCollections.call(work_form: @work_form)
+      @move_form = Admin::MoveForm.new(content_id: params[:content_id])
+      render :form
     end
 
-    def create
+    def create # rubocop:disable Metrics/AbcSize
       authorize!
 
-      collection = Collection.find_by(druid: params[:collection_druid])
-      Admin::Move.call(work_form: @work_form, work: @work, collection:, version_status: @version_status)
-
-      flash[:success] = I18n.t('messages.work_moved')
-      redirect_to wait_works_path(@work.id)
+      @move_form = Admin::MoveForm.new(work_form:, **move_params)
+
+      if @move_form.valid?
+        work_form.content_id = @move_form.content_id
+        Admin::Move.call(work_form:, work:, collection: @move_form.collection, version_status:)
+
+        flash[:success] = I18n.t('messages.work_moved')
+        # This breaks out of the turbo frame.
+        respond_to do |format|
+          format.turbo_stream do
+            render turbo_stream: turbo_stream.action(:full_page_redirect, wait_works_path(work.id))
+          end
+        end
+      else
+        render :form, status: :unprocessable_entity
+      end
```

Comment:
Minor tweak: how would you feel about shifting the `else` logic into a guard clause? Why? It removes one level of indentation from the `if` logic. I generally find code that is indented multiple levels to be harder to grok, because you have to hold in your head what each level of indentation means. Shooting for more "vertical" code than "horizontal."

```suggestion
      return render :form, status: :unprocessable_entity unless @move_form.valid?

      work_form.content_id = @move_form.content_id
      Admin::Move.call(work_form:, work:, collection: @move_form.collection, version_status:)

      flash[:success] = I18n.t('messages.work_moved')
      # This breaks out of the turbo frame.
      respond_to do |format|
        format.turbo_stream do
          render turbo_stream: turbo_stream.action(:full_page_redirect, wait_works_path(work.id))
        end
      end
```

Reply:
All the other `if ...valid?` blocks in the controllers are like 2-3 lines. This `if` block is a bit longer and gets up to four levels of indentation relative to the method. 

Patternwise, I'm not too worried about us not understanding what `unless ...valid?` means. My sense is that we use one or the other depending on context. In general, I'd say we prefer guard clauses to indentation where we can justify it. I think we can make a case that it makes sense here.